### DISCUSSION TITLE
Not receiving client reply WhatsApp Cloud

### DISCUSSION THREAD

**Discussion by adbasanta** [chatwoot/chatwoot#6157] [Self hosted] [answered]
Does anybody have experienced the same issue as me? I can send a message template but I cannot receive any response from the client as per the image below. Do I need to set anything in the .env file? Thanks for helping.
![image](https://user-images.githubusercontent.com/89345000/210161379-35dad9a3-c7ea-4cce-8e68-ad0740f7145d.png)

**Comment by pranavrajs** [comment]
@adbasanta Have setup the webhooks as per the doc here https://www.chatwoot.com/docs/product/channels/whatsapp/whatsapp-cloud#enable-the-webhook and subscribed to the messages event?

**Reply by adbasanta** [reply]
@pranavrajs thanks a lot for the reply.
Yes, I did the webhook and token in the Facebook developer app as per below image. The values I put in there are the ones being displayed at the end of Inbox creation. I also did subscribe to the webhook fields. I then tried to send a test message and successfully received it as per the below image but not with a reply from real numbers.
I also cannot receive any new messages from any numbers.
![webhook snapshot](https://user-images.githubusercontent.com/89345000/210195480-ab641888-3f5e-48f4-be0a-6cc22e519c03.PNG)
![test img snapshot](https://user-images.githubusercontent.com/89345000/210195487-dbfc7db8-35a4-41dd-a804-01bec1d0329e.PNG)

**Reply by pranavrajs** [reply]
@adbasanta Are you using a test number or the production number?

**Comment by adbasanta** [comment]
@pranavrajs I am using the production number as per below.
![production number](https://user-images.githubusercontent.com/89345000/210200136-2536b289-b23c-479a-a6c2-298b539a1174.PNG)

I wonder if my setup is causing it?
My main domain is unencrypted (no SSL just plain http)
The subdomain that I used was auto-encrypted by Lets Encrypt during chatwoot setup
This subdomain is pointing to my noip DNS (no SSL) which points to my self-hosted machine where chatwoot is running

**Comment by adbasanta** [comment, marked as answer]
I found out that the problem is in the Meta Business Settings in Phone Numbers. I just remove the phone number there then added it again and it worked. Thank you!
![phone numbers](https://user-images.githubusercontent.com/89345000/210319306-385d46f9-4328-46ef-96c9-49e00f2e5821.PNG)
